app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt

```diff
@@ -126,4 +131,22 @@ class MoneyActivity : SimpleActivity(), Calculator , TaxCalculator {
 
     private fun getMoneyButtonIds() = arrayOf(btn_tip, btn_currency, btn_taxes)
     private fun getButtonIds() = arrayOf(btn_decimal, btn_0, btn_1, btn_2, btn_3, btn_4, btn_5, btn_6, btn_7, btn_8, btn_9)
-}
+
+
+    private fun displayTipsDialog() {
+        val numbers = arrayOf("10%", "12%", "15%", "18%", "20%", "25%")
+
+        val tipDialog = AlertDialog.Builder(this@MoneyActivity)
+        tipDialog.setTitle("Calculate Tip")
+
+        tipDialog.setItems(numbers, DialogInterface.OnClickListener { _, index ->
+            var tipPercentage = numbers[index].trimEnd('%').toDouble() / 100
+            calc.calculateTip(tipPercentage)
+        })
+
+        tipDialog.setNegativeButton("Cancel", DialogInterface.OnClickListener { _, _ ->
```

Comment:
I think you could just leave a { } instead of the empty lambda

Reply:
Thanks

Reply:
Unfortunately, it doesn't work if I try putting '{ }' instead of the lambda function. It seems that's just how Kotlin code is written.